bin/yarn-install

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+require('shelljs/global')
+
+exec('cd cli && yarn && cd ..')
```

Comment:
I'm not sure if we should use `yarn` or `npm` here. Using `yarn` would require the user to download another dependency, and there could be issues because we're installing `exosphere-sdk` via `npm` but the subproject dependencies via `yarn`. Although I've installed `exosphere-sdk` on my machine when the `post-install` script was using `yarn`, and at least the commands `exo-setup`, `exo-run`, and `exo-deploy` work.

On the other hand, I tried installing `exosphere-sdk` locally when all these commands were using `npm` instead of `yarn`, and it was extremely slow... 🐢